### PR TITLE
Collapsible Dataset Info

### PR DESCRIPTION
**Pull request by rabitt**
- [x] Top level loader docs: collapsible dataset info block
- [x] make contributing code examples collapsible

### CODE REVIEW

**Comment by rabitt** [comment]
Should be ready to merge, but if anyone sees an issue feel free to just fix it and merge :)

**Review by magdalenafuentes — APPROVED** [review]

**Review by magdalenafuentes — COMMENTED** [review]
I love how the collapsible blocks look like! Made few nit comments and modifications. 

By looking at the code I guess it's not so easy to collapse attributes and functions right?

**Comment by magdalenafuentes** [comment]
See my comments on a few details on the docs and after solving those we can merge!

**Comment by magdalenafuentes** [comment]
I'm addressing #388 here because there are two TODOs in this PR: `overview.rst L136` which I'm not sure what to do (will leave a proposal here, @rabitt feel free to check and modify), and `quick_reference.rst L221` which I'll remove as we discussed yesterday. I'll do that in a separated PR

**Comment by magdalenafuentes** [comment]
So after merging this closes #388

**Comment by rabitt** [comment]
> By looking at the code I guess it's not so easy to collapse attributes and functions right?

I thiiink that could be controlled by changing the template overall but I didnt find a good way to do it. If we want to explore that we can, but this should be complementary

File: docs/source/mirdata.rst
Comment (rabitt) [reply docs/source/mirdata.rst:4]: ah ok! I'd seen "datasets" as cleaner but you're right! I'll change it back

File: docs/source/overview.rst
Comment (rabitt) [reply docs/source/overview.rst:8]: I tried to put it just below. Is there a part of this paragraph that's not below that you think is important for people to know?

**Comment by genisplaja** [comment]
That's great!! Should we make code examples from `tutorial.rst` collapsible like `contributing.rst` as well? In that case, should we first merge #378 and then apply the collapsible extension to `tutorial.rst`?

File: docs/source/overview.rst
Comment (rabitt) [reply docs/source/overview.rst:27]: its in the datasets too! I put this bullet here because i noticed some people using mirdata to download but still writing their own load functions. Even though we expose the loaded data directly through the attributes I wanted to make it super clear that we wrote these functions for people

Comment (magdalenafuentes) [reply docs/source/overview.rst:29]: True! You can directly use the annotations path, forgot :) That should be in the `work datasets remotely` section of the tutorial IMO @genisplaja

**Comment by magdalenafuentes** [comment]
+1 to collapsing the tutorials :)

File: docs/source/overview.rst
Comment (genisplaja) [reply docs/source/overview.rst:29]: Yes! There is a section called `Accessing annotations remotely` where there's a code example of that. I can give it another title if you want!

Comment (rabitt) [reply docs/source/overview.rst:29]: I agree it's more about remote access, but I wanted something up-front in the dataset description so that people don't miss the point that we're providing functions for actually loading the annotations files. I realize it's not the main use case, and I agree it's good to talk about it in more detail as a "using remote data" feature. I don't have another idea for how to clearly say "we have functions for loading specific annotations you don't need to write them yourself" - do you have an idea for how?

Comment (genisplaja) [reply docs/source/overview.rst:29]: What about something like that:
```
``load_*()``: load dataset's files (audio, metadata, annotations, etc.) into standard formats so additional functions are not needed
```

Comment (rabitt) [reply docs/source/overview.rst:29]: @magdalenafuentes I'll update with Pedro's suggestion and merge, and we can adjust further as needed!

**Pull request merged**
